### ISSUE TITLE
0.17 release

### ISSUE THREAD

**Issue by adamralph** [adamralph/scriptcs-nancy#117]
**Ready** when all other issues forming part of the release are **Done**.

- [ ] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [ ] check build, create release in [GitHub UI](https://github.com/adamralph/scriptcs-nancy/releases) including releaseNotes, mentioning non-owner contributors, if any
- [ ] push nuget package
- [ ] copy release notes from GitHub to NuGet
- [ ] de-list pre-relse nuget packages if present
- [ ] tweet, mentioning contributors and post link for easy retweeting ;-)
- [ ] post tweet in JabbR (Nancy, scriptcs, general-chat)
- [ ] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [ ] increment minor version
- [ ] change version in sample/packages.config
- [ ] push to origin branch, create PR to upstream master
- [ ] check build, merge PR
- [ ] create a new milestone for the next release
- [ ] create new issue based on this one for next release, adding it to the new milestone
- [ ] close all issues on this milestone
- [ ] close this milestone
